pkg/reconciler/v1alpha1/revision/queueing_test.go

```diff
@@ -255,10 +262,10 @@ func TestNewRevisionCallsSyncHandler(t *testing.T) {
 	defer close(stopCh)
 
 	kubeInformer.Start(stopCh)
-	buildInformer.Start(stopCh)
 	servingInformer.Start(stopCh)
 	servingSystemInformer.Start(stopCh)
 	vpaInformer.Start(stopCh)
+	buildInformerFactory.StopChannel = stopCh
```

Comment:
Can we hoist the definition of `stopCh` to assign it as part of initialization above?